zk_prover/examples/gen_inclusion_verifier.rs

```diff
@@ -0,0 +1,71 @@
+#![feature(generic_const_exprs)]
+
+use serde_json::to_string_pretty;
+use snark_verifier_sdk::{
+    evm::{evm_verify, gen_evm_proof_shplonk, gen_evm_verifier_shplonk},
+    CircuitExt,
+};
+use std::{fs::File, io::Write, path::Path};
+use summa_solvency::{
+    circuits::{
+        merkle_sum_tree::MstInclusionCircuit,
+        types::ProofSolidityCallData,
+        utils::{
+            gen_proof_solidity_calldata, generate_setup_artifacts, write_verifier_sol_from_yul,
+        },
+    },
+    merkle_sum_tree::MerkleSumTree,
+};
+
+const LEVELS: usize = 4;
+const N_ASSETS: usize = 2;
+const N_BYTES: usize = 14;
+
+fn main() {
```

Comment:
Now that I see this function again, I realize that it might be a bit confusing since to generate the verifier you don't really need to instantiate the circuit with a `csv` input but you can use the `init_empty()` API.

Conversely, you need the `csv` file input when generating the test proof.

I'll open a separed issue for this